README.md

```diff
@@ -43,11 +44,7 @@
 - ### If using BLE Connection
   1. Download SwitchBot App on App Store or Google Play Store
   2. Register a SwitchBot account and log in into your account
-  3. Click on Device wanting to connect too plugin
-     - Click the Settings Gear
-     - Click Device Info
-     - Copy BLE Mac aka `deviceId`
-  4. Input your `deviceId` into the Device Config
+  3. Input your `deviceId` into the Device Config
```

Comment:
Thanks for the PR!

We would need to flag this as optional, as it is not required but helpful if you want to override defaults.